libwacom/libwacom-database.c

```diff
@@ -295,32 +419,65 @@ libwacom_parse_stylus_keyfile(WacomDeviceDatabase *db, const char *path)
 
 	groups = g_key_file_get_groups (keyfile, NULL);
 	for (i = 0; groups[i]; i++) {
-		WacomStylus *stylus;
-		WacomStylusId id;
+		WacomStylus *stylus, *other;
+		WacomStylusId id, lookup;
```

Comment:
Move `lookup` into the condition where we use it please, makes the code more obvious.